client/packages/invoices/src/Prescriptions/DetailView/SidePanel/AdditionalInfoSection.tsx

```diff
@@ -27,6 +28,8 @@ export const AdditionalInfoSectionComponent: FC = () => {
   const [commentBuffer, setCommentBuffer] = useBufferState(comment ?? '');
   const { localisedDate } = useFormatDateTime();
 
+  const debouncedUpdate = useDebouncedValueCallback(update, [commentBuffer]);
```

Comment:
```suggestion
  const debouncedUpdate = useDebouncedValueCallback(update, [commentBuffer], 1000);
```

Your change is still causing numerous API calls.. 

https://github.com/user-attachments/assets/45fe62c8-220c-4f31-84b8-0172223b632b

If you look at examples where the comment is not being jumpy, you will see that there is a 1k timeout on the API call.. looks like the 500 timeout on the `useDebouncedValueCallback` is not enough and still causes constant API calls while throttlin' network to 3G....

Reply:
@roxy-dao thanks for catching that! For some reason when I tested previously it wasn't behaving strangely. But I'm still finding that using ```useBufferState``` with the ```debouncedValueCallback``` even with 1000 timeout doesn't fully resolve the issue. I'm finding that using ```useState``` instead seems to be working fine. Is it ok to swap?

Reply:
If extending the timeout to 1k causes problems then your original fix definitely caused problems...? useBuffer is saving the value into state.... maybe needs some thinking?

Reply:
Oh my original fix did cause problems. It's just for whatever reason when I tested before putting the PR in it was working and so I didn't catch it then. When I test now it is causing problems. UseState isn't but I'll have a look at how it's being used in other places.